src/pages/library/Code.test.js

```diff
@@ -35,48 +37,14 @@ describe('Code page', () => {
       },
     ];
 
-    const mockOidcUser = {
-      id_token: undefined,
-      session_state: null,
-      access_token: token,
-      refresh_token: '',
-      token_type: '',
-      scope: undefined,
-      profile: {
-        sub: '',
-        iss: '',
-        aud: '',
-        exp: 0,
-        iat: 0,
-      },
-      expires_at: undefined,
-      state: undefined,
-      expires_in: 0,
-      expired: undefined,
-      scopes: [],
-      toStorageString: '',
-    };
-
-    Ajax.mockImplementation(() => {
```

Comment:
This must be a left-over from an earlier version of this test... Ajax.mockImplementation is called a 2nd time just after this, which negates this mock implementation completely.  So this block and its referenced bits like mockOidcUser can be removed.